pkg/documents/documents.go

```diff
@@ -246,6 +264,24 @@ func NewDocumentRepo(logger log.Logger, db *sql.DB) DocumentRepository {
 		logger: logger,
 	}
 }
+
+func (r *sqlDocumentRepository) findActiveDocument(customerID string, documentID string, namespace string) error {
+	query := `select documents.document_id from documents
+inner join customers on customers.namespace = ? where documents.customer_id = ? and documents.document_id = ? and documents.deleted_at is null
+limit 1;`
+	stmt, err := r.db.Prepare(query)
+	if err != nil {
+		return fmt.Errorf("getCustomerDocuments: prepare %v", err)
```

Comment:
I recommend not putting function names in error messages, something like:
`fmt.Errorf("finding active document: %v", err)` would work well